### PR TITLE
Fix cavs HDA topologies

### PR DESCRIPTION
**Pull request by ranj063**

### CODE REVIEW

File: tools/topology/topology2/cavs/cavs-passthrough-hda.conf [hunk 1]
```diff
@@ -16,7 +16,7 @@ Object.Dai {
 			stream_name 'Analog Playback and Capture'
 			period_sink_count 0
 			period_source_count 2
-			format s32le
+			format s16le
```
Comment (plbossart) [inline tools/topology/topology2/cavs/cavs-passthrough-hda.conf:19]: ?? Why would we restrict the format to 16bits?

File: tools/topology/topology2/include/common/tokens.conf [hunk 2]
```diff
@@ -134,7 +134,7 @@ Object.Base.VendorToken {
 		out_ch_cfg		1935
 		out_interleaving_style	1936
 		out_fmt_cfg		1937
-		out_sample_type		1917
+		out_sample_type		1938
```
Comment (plbossart) [inline tools/topology/topology2/include/common/tokens.conf:137]: ouch

**Review by plbossart — COMMENTED** [review]
the token fix is fine, the change to 16 bits less clear.

File: tools/topology/topology2/cavs/cavs-passthrough-hda.conf (hunk 1 above)
Comment (ranj063) [reply tools/topology/topology2/cavs/cavs-passthrough-hda.conf:19]: I'm going to drop the first patch and replace it to remove the format entirely @plbossart. It is never really used in the kernel or the fw

**Review by plbossart — APPROVED** [review]
nice catches @ranj063

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**